examples/usbboot.ts

```diff
@@ -13,19 +13,61 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { argv } from 'process';
+
+/**
+ * Unlocking with secureboot
+ *
+ * If you need to expose the internal storage of a cm4 with secureboot, you need to provide a signed boot-image to set the device in MSD mode.
```

Comment:
nitpick: I would remove cm4 here too and replace with a generic `device`

Reply:
This is specifically designed for the CM4 and we don't know if it would work on other pi.

Reply:
We know it will work for the RPI4 - and with the information we have the RPI5 mechanism is similar. A generic comment just indicates that the mechanism is intended to be used for secured devices. Leaving cm4 will mean that if it just works for RPI4 no-one will ever bother changing the incorrect comment.

We could specify in the commit log that it was tested on the CM4 only.

But I am just nitpicking - it's just a comment so do as you prefer.